modules: add new missing screenshots

lasers opened the pull request:
Nine-plus (9+) screenshots.
* 🖼 🖼 🖼
* 🖼 🖼 🖼
* 🖼 🖼 🖼 ➕

tobes commented on py3status/modules/weather_owm.py:
```diff
@@ -233,24 +233,28 @@
+{'full_text': '🌫 ○ 59°F, foggy ⛅ ☼ 🌧'}
+
+# Currently foggy, 59°F outside, with forecast of cloudy
+# tomorrow, sunny the next day, then rainy in two days.
```
I don't trust these comments here.

tobes commented on py3status/modules/weather_owm.py:
```diff
@@ -233,24 +233,28 @@
-  api_key = '...'
-
-  icons {
+  icons = {
```
This is should not be part of this PR as it is not related to screenshots.

I will accept it but you need to make your PRs cleaner.  If you do this then they will get actioned far faster.  Good catch though.

tobes commented on py3status/modules/graphite.py:
```diff
@@ -66,6 +66,9 @@
 @author ultrabug
+
+SAMPLE OUTPUT
+{'color': 'full_text': 'graphite'}
```
I'm not convinced by this output.  I may try to get some real data

ultrabug replied:
yeah switch to '412 req/s' instead as graphite is meant to return numbers with units usually

lasers commented:
>Can you confirm if you have built the documentation using this branch.

Can't. I'm missing `fonttools` on `Solus`.

tobes commented:
> >  Can you confirm if you have built the documentation using this branch.
>
> Can't. I'm missing fonttools on Solus.

Ok well I'll see if I can do a test build at some point.  I'm pretty sure your comments will break stuff.

Is this all faked output?  Or is some of it real - as in actually produced by the module

lasers commented:
I faked everything. I looked at this again.

In `systemd`, the real is probably `sshd.service: active`. Fixed.

`graphite` is not `graphite`. :-\

`icinga2` is missing at least `4: INVALID STATUS`. Not sure if we want to add error code too.... Also, this could be superseded by `getjson`.

I think I'm right on target with `glpi`, `ns_checker`.

In `pingdom`, I'm sure it is `{server_name}: 323ms` and if more than one, then `{server_name}: 322ms, {server_name}: 341ms`. I use `Pingdom` as server name. Maybe we should use `py3status` ? Nah. 

I vaguely recall `thunderbird_calendar` is what it is, but we should merge `thunderbird_todos` anytime before the release to avoid any depreciation. So no worries.

I removed the comments because you don't trust the weather predictions.

ultrabug commented:
just change graphite and let's go

lasers commented:
`graphite` updated.

tobes commented:
@lasers this PR has broken screenshots (I'll fix them) can you please try to test your PRs before you make them?  You need to take much more care with your code.

lasers commented:
>Can you confirm if you have built the documentation using this branch.
>>Can't. I'm missing fonttools on Solus.

I will try to switch a distro so I can test the docs. I need to be more careful with amend tweaks (wifi).

tobes commented:
> I need to be more careful with amend tweaks

You do.  As long as you learn from these things then all is good.  This is also why you should make smaller more focused PRs as they are easier to review and catch these issues
